sstmac/hardware/switch/network_switch.h

```diff
@@ -82,9 +82,9 @@ class NetworkSwitch :
   SST_ELI_DECLARE_DEFAULT_INFO()
   SST_ELI_DECLARE_CTOR(uint32_t,SST::Params&)
 
-  virtual void init(unsigned int phase);
+  void init(unsigned int phase) override;
 
-  virtual ~NetworkSwitch();
+  ~NetworkSwitch();
```

Comment:
Doesn't this still need to be virtual in case I delete a type of `NetworkSwitch*`?